pkgs/os-specific/linux/amdgpu-pro/patches/0005-pci.patch

```diff
@@ -0,0 +1,20 @@
+--- a/amd/amdkcl/kcl_pci.c	2018-09-17 11:48:14.257542945 -0500
++++ b/amd/amdkcl/kcl_pci.c	2018-09-17 12:18:37.921341917 -0500
+@@ -1,7 +1,7 @@
+ #include <kcl/kcl_pci.h>
+ #include <linux/version.h>
+ 
+-#if defined(BUILD_AS_DKMS)
++#if defined(BUILD_AS_DKMS) && LINUX_VERSION_CODE < KERNEL_VERSION(4, 16, 0)
+ 
+ /**
+  * pci_enable_atomic_ops_to_root - enable AtomicOp requests to root port
+@@ -14,7 +14,7 @@
+  * and/or 128-bit AtomicOp completion), or negative otherwise.
+  *
+  */
+-int pci_enable_atomic_ops_to_root(struct pci_dev *dev, u32 comp_caps)
++int _kcl_pci_enable_atomic_ops_to_root(struct pci_dev *dev, u32 comp_caps)
```

Comment:
You shouldn't need to rename this.  It should still have the same name with older kernels.

Reply:
Yeah, oops, didn't actually mean to push that. :)